initfiles/componentfiles/configschema/xsd/dafilesrv.xsd

```diff
@@ -1,34 +1,31 @@
-        </xs:attributeGroup>
-        <xs:sequence>
-            <xs:element name="DafilesrvProcess" hpcc:itemType="dafilesrv" minOccurs="1" maxOccurs="1" hpcc:class="valueSet" hpcc:docid="daFs.t">
-                <xs:complexType>
-                    <xs:sequence>
-                        <xs:element name="Instance" maxOccurs="unbounded" hpcc:class="elementSet" hpcc:displayName="Bound Computers">
-                            <xs:complexType>
-                                <xs:attributeGroup ref="computerNodeReference"/>
-                                <xs:attribute name="directory" type="xs:string" use="optional" hpcc:presetValue="c$\dafilesrv" hpcc:hidden="true"/>
-                                <xs:attributeGroup ref="executionSettings"/>
-                            </xs:complexType>
-                        </xs:element>
-                    </xs:sequence>
-                    <xs:attributeGroup ref="buildInfo"/>
-                    <xs:attribute name="name" type="xs:string" use="required" hpcc:tooltip="Name for this process"/>
-                    <xs:attribute name="description" type="xs:string" use="optional" hpcc:presetValue="DaFileSrv process" hpcc:tooltip="Description for this process"/>
-                    <xs:attribute name="version" type="xs:string" use="optional" hpcc:presetValue="1" hpcc:tooltip="Version identifier used to select which process will be started"/>
-                    <xs:attributeGroup ref="executionSettings"/>
-                </xs:complexType>
-            </xs:element>
-        </xs:sequence>
-
-    </xs:complexType>
+    <hpcc:insert hpcc:schemaPath="/Environment/Software">
+        <xs:element name="DafilesrvProcess" hpcc:itemType="dafilesrv" minOccurs="1" maxOccurs="1" hpcc:class="valueSet" hpcc:docid="daFs.t">
+            <xs:complexType>
+                <xs:attributeGroup name="executionSettings">
+                    <xs:attribute name="parallelRequestLimit" type="percent" use="optional" hpcc:presetValue="20" hpcc:tooltip="Defines the maximum number of concurrent dafilesrv requests allowed. Requests that exceed the limit will be delayed. A value of 0 disables throttling. Overrides global settting."/>
+                    <xs:attribute name="throttleDelayMs" type="xs:nonNegativeInteger" use="optional" hpcc:presetValue="5000" hpcc:tooltip="Defines how many milliseconds delayed requests will be delayed by. Overrides global settting."/>
+                    <xs:attribute name="throttleCPULimit" type="xs:nonNegativeInteger" use="optional" hpcc:presetValue="75" hpcc:tooltip="if after the initial delay, the CPU % falls below this setting, the transaction will be allowed to continue, i.e. the limit can be exceeded this way. Overrides global settting."/>
```

Comment:
Many of those xsd lines are too long to read without using mouse to move back and forth. Can you break one line into multiple lines if the line length > 120?

Reply:
I have heard that comment before. One thing I was trying to do was to keep the XSD files shorter in length. I will go ahead and reformat them to have shorter line lengths.